### PR TITLE
Tweaking the clone-settings-php command so it doesn't choke horribly when using Drush 6

### PR DESCRIPTION
**Pull request by blakehall**
The output of drush status has changed, so we need to be a little smarter about finding the site path to find the settings.php file to clone.

### CODE REVIEW

File: jgd.drush.inc [hunk 1]
```diff
@@ -173,15 +173,23 @@ function drush_jgd_clone_settings_php($site_alias, $prefix = NULL) {
   // Load up the status stuff so we can get the conf_path(). Wish there was
   // another way of doing this.
   $results = drush_invoke_process($source_record, 'status', array(), array(), array('integrate' => FALSE));
+  $drush_version = drush_core_version();
+
+  if (strpos($drush_version, '5') === 0) {
```
Comment (q0rban) [inline jgd.drush.inc]: Seems like this should check if the version is less than 6, not make sure it's 5.

Comment (blakehall) [inline jgd.drush.inc]: That was basically my thinking by using 6's default in the else statement.

Comment (q0rban) [inline jgd.drush.inc]: I'm guessing that in Drush 4, it should also use 'Site path', yet this code would be using 'site' for anything other than Drush 5. I would say that probably doesn't matter, except that MSLO is currently using Drush 4.

Comment (blakehall) [inline jgd.drush.inc]: Fair enough, that makes sense.

I'll see what I can do about getting it working with punch cards too :wink:

**Comment by davereid** [comment]
Might be worth switching to `drush dd @alias:%site` which works in both Drush 5 and 6 and gives you the full path to that site directory.

**Pull request closed without merging**